src/actors/builtin/storage_miner/storage_miner_actor_code.go

```diff
@@ -63,7 +64,7 @@ func (a *StorageMinerActorCode_I) OnSurprisePoStChallenge(rt Runtime) {
 	challengedSectorsRandomness := rt.RandomnessWithAuxSeed(
 		filcrypto.DomainSeparationTag_SurprisePoStSampleSectors,
 		rt.CurrEpoch()-node_base.SPC_LOOKBACK_POST,
-		addr.Serialize_Address_Compact(rt.CurrReceiver()),
+		rt.CurrReceiver().Bytes(),
```

Comment:
Inputs to randomness are also defined to be CBOR, I believe. @sternhenri ?

Reply:
I have changed this to use CBOR, is this comment merge blocking?

Reply:
Picking this back up in #775. Simplified randomness for lotus purposes there